deeppavlov/utils/server/server.py

```diff
@@ -31,7 +32,18 @@
 
 SERVER_CONFIG_FILENAME = 'server_config.json'
 
+
+class PollerFilter(Filter):
```

Comment:
A docstring for this class would be great

Reply:
Docstring has added